cmd/main.go

```diff
@@ -139,8 +148,8 @@ func main() {
 	}
 	if namespace != "" && nodeName != "" && scConfig.TransportHost.Type == common.HTTP {
 		// if consumer doesn't pass namespace then this will default to empty dir
-		if e := client.InitConfigMap(scConfig.APIVersion, scConfig.StorePath, nodeName, namespace); e != nil {
-			log.Errorf("failed to initlialize configmap, subcription will be stored in empty dir %s", e.Error())
+		if e := client.InitConfigMap(scConfig.APIVersion, scConfig.StorePath, nodeName, namespace, configMapRetryInterval, configMapRetryCount); e != nil {
```

Comment:
this bug is loss off subscription on restart of pods. Configmap won't be deleted when restarted,